src/engraving/libmscore/textbase.cpp

```diff
@@ -3084,6 +3092,17 @@ Sid TextBase::offsetSid() const
     return Sid::NOSTYLE;
 }
 
+bool TextBase::moveCursor(TextCursor* cursor, int key, bool ctrlPressed, TextCursor::MoveMode moveMode) const
```

Comment:
But this already works? (using Ctrl+left/right to move words).  Any reason you needed to add this?

Reply:
This is done so the individual elements can modify the movement behavior of the cursor if required. (TempoText does not allow the cursor to move inside the tempo marking itself)

Reply:
Are you saying somebody can't, for example, edit the = symbol and replace it with ≈? That would be a serious regression on MU3 if so...

Reply:
@Tantacrul should probably answer this

Reply:
As long as I can have this: 
![image](https://user-images.githubusercontent.com/5364225/129537357-2abfca24-f2cb-4b8a-b2e8-8fce6f3885e3.png)